_posts/2025-06-18-hibernate7-on-quarkus.adoc

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: 'Hibernate 7 on Quarkus: every new version is a better database experience for the users'
+date: 2025-06-18
+tags: hibernate, database, jpa
+synopsis: A roundup on the new upgrades of Hibernate on Quarkus.
+author: lmolteni
+---
+:imagesdir: /assets/images/posts/hibernate7
+ifdef::env-github,env-browser,env-vscode[:imagesdir: ../assets/images/posts/hibernate7]
+
+=== Introduction
+Hibernate is improving at a very fast speed, and so is its Quarkus support, as great database access is a key part of the Quarkus experience.
+The new Quarkus 3.24 release upgrades Hibernate to version 7, a major upgrade that implies some breaking changes, and thus will require some investigation with the https://docs.jboss.org/hibernate/orm/7.0/migration-guide/migration-guide.html[migration guide] when upgrading.
+Developers working on Hibernate and Quarkus are constantly collaborating, so here’s a quick peek at what has happened in the last months and what Quarkus users might expect in the future.
+
+=== License and Governance Updates 
+Both Quarkus and Hibernate are now projects of the https://www.commonhaus.org[Commonhaus Foundation], a non-profit organization dedicated to creating a collaborative environment for open-source libraries.
+Since the upgrade to Hibernate 7, Quarkus and all modules of Hibernate now share the same open-source license: the https://www.apache.org/licenses/LICENSE-2.0[Apache License Version 2.0]. Implementing a proper open-source approach is crucial to the success of the two projects.
```

Comment:
```suggestion
Since the upgrade to Hibernate 7, Quarkus and all modules of Hibernate now share the same open-source license: the https://www.apache.org/licenses/LICENSE-2.0[Apache License Version 2.0].
```

(Kinda agree with this, in the spirit of "show, don't tell")